src/docs/components/styling.md

```diff
@@ -275,6 +275,8 @@ This file must be manually imported in the `index.html` of your application.
 
 ## IE support
 
+> Starting with Stencil v3.0.0, support for IE 11 has been marked as deprecated and will be removed in the next major version of Stencil.
```

Comment:
Not really a comment on this PR _per se_, but will be nice to standardize admonitions and [format them consistently w/ Docusaurus](https://docusaurus.io/docs/markdown-features/admonitions). Just took a look at https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Aionic-team%2Fstencil-site+path%3A*.md+%2F%5E%3E%2F and we sometimes start them with `NOTE:`, `⚠️ Warning`, `Note`, etc and sometimes we just go right into the text. Kinda messy! For now I think it's best just to do an unadorned blockquote, and we'll convert these to `info`, `caution`, `note`, etc later.

Reply:
Yeah, this might be a quick update we have to make between now and Wednesday. We'll see how this all shakes out 😆